app/templates/activity.html

```diff
@@ -46,6 +46,12 @@
     <p>{{ user_link(event.user) }} shared:</p>
 
     {{ event.message }}
+  {% elif event.type == 'simple_user_event' %}
+    {% if event.subtype == event.SUBTYPE_USER_JOINED %}
+      <p>{{ user_link(event.user) }} joined NOI.</p>
```

Comment:
Er, this isn't internationalized, but neither are the other messages in the activity stream, yet... Might be easier to just file a separate issue for internationalizing the stream, for now...